### PR TITLE
Update README: instructions on updating wx and building on Ubuntu

### PR DESCRIPTION
**Pull request by niauah**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,6 +81,62 @@ wxWidgets is located in vendor/wxWidgets, and can be built like so:
...
+Clone the repo from git (use PR #134)
+```
+git clone git@github.com:Aegisub/Aegisub.git
+```
+Then run
+```
+cd Aegisub/
+sudo apt-get install  autoconf autopoint automake libass-dev luajit intltool -y
+./autogen.sh
+./configure
```
Comment (kinow) [inline README.md]: This failed on my Ubuntu due to missing `libboost-chrono-dev`. 

Now it's failing with `configure: error: Could not link against boost_locale !`, but I am still investigating why.

Comment (wangqr) [reply README.md]: Have you installed libboost? Aegisub depends on `libboost-{chrono,filesystem,locale,regex,system,thread}-dev`

**Comment by kinow** [comment]
Juat the dev I think, and thread. Will try installing everything then.

**Comment by niauah** [comment]
merged into #165

**Pull request closed without merging**